libiocage/lib/Jail.py

```diff
@@ -390,7 +390,7 @@ def create(self, release_name):
 
         backend = None
 
-        is_basejail = self.config["type"] == "basejail"
+        is_basejail = self.config["basejail"] == True
```

Comment:
This should be `is_basejail = self.config.get("basejail", True)` as that's pythonic and much nicer to read.

Reply:
i did is True, flake8 could've done a better job suggesting what you suggested :D

Reply:
n.b.: in other places we already use

```python
    @property
    def basejail_backend(self):

        if self.config["basejail"] is False:
            return None

        if self.config["basejail_type"] == "nullfs":
            return libiocage.lib.NullFSBasejailStorage.NullFSBasejailStorage

        if self.config["basejail_type"] == "zfs":
            return libiocage.lib.ZFSBasejailStorage.ZFSBasejailStorage

        return None
```